rules/linux/lnx_system_net_disc_firewall_enum.yml

```diff
@@ -0,0 +1,31 @@
+title: System Network Discovery - Firewall Enumeration
+id: 71da9e5a-fb1e-46a8-abc1-28c80173af4c
+status: experimental
+description: Detects enumeration of firewall configuration
+author: remotephone, oscd.community
+date: 2020/10/06
+references:
+    - https://github.com/redcanaryco/atomic-red-team/blob/master/atomics/T1016/T1016.md
+    - https://attack.mitre.org/techniques/T1016
```

Comment:
could you please remove a link to the ATT&CK, as it was already mentioned in `tags`?